projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -157,12 +157,23 @@ public static function load_etk_features() {
 			return;
 		}
 
+		if ( ! function_exists( 'is_plugin_active' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+
 		require_once __DIR__ . '/features/block-editor/custom-line-height.php';
 		require_once __DIR__ . '/features/block-inserter-modifications/block-inserter-modifications.php';
 		require_once __DIR__ . '/features/hide-homepage-title/hide-homepage-title.php';
 		require_once __DIR__ . '/features/jetpack-global-styles/class-global-styles.php';
 		require_once __DIR__ . '/features/mailerlite/subscriber-popup.php';
-		require_once __DIR__ . '/features/newspack-blocks/index.php';
+
+		/**
+		 * It has a conflict with newspack or newspack-blocks plugin. If it's active, don't load the newspack feature.
+		 */
+		if ( ! ( is_plugin_active( 'newspack-plugin/newspack.php' ) || is_plugin_active( 'newspack-blocks/newspack-blocks.php' ) ) ) {
```

Comment:
This resolves the error as it was experienced but doesn't protect against the possibility of newspack moving the class to a new plugin (similar to how it is now part of mu-wpcom). 

A more future proof approach would be to only include newspack-blocks/index after plugins loaded that way you can do an explicit check for the newspack-blocks class existing vs a check on if plugins are active. That depends on if there are any dependencies on the code being available earlier.

Reply:
Yes! However, there is an issue that the feature will be loaded via the ETK plugin (we're still removing the codebase) if we load the newspack blocks feature on the `plugin_loaded` hook. The reason is we use the flag defined in `newspack-blocks/index` to turn off the feature in the ETK plugin, so the file should be loaded before the ETK plugin.

> A more future proof approach would be to only include newspack-blocks/index after plugins loaded that way you can do an explicit check for the newspack-blocks class existing vs a check on if plugins are active.

I'm afraid that the version of the newspack-blocks plugin might not be compatible with the feature here. As a result, I'd prefer to disable the feature because it's the current behavior that we deactivate the ETK plugin if the newspack-blocks plugin is active.

Reply:
I'm proposing https://github.com/Automattic/jetpack/commit/603956817518c42bcf09da28a60b96d780489dae to resolve this issue as follows:
- Load the feature when the plugin is loaded and only load it in the editor or frontend page to avoid the issue of the activating the newspack blocks plugin
- Check whether the class is declared before loading the feature